src/components/accessories/patientExams/patientExamsTable/PatientExamsTable.tsx

```diff
@@ -114,7 +114,7 @@ const PatientExamsTable: FunctionComponent<IOwnProps> = ({
           <InfoBox type="warning" message={t("common.emptydata")} />
         </div>
       )}
-      {labStatus === "LOADING" && (
+      {labStatus === "IDLE" && (
```

Comment:
Uhm what's about the loading status? I think that the issue is inside the labStatus value and not how it's rendered.

Reply:
I have committed without changing it, it is a mistake. I correct it now